tests_fp/solidity/test_add_liquidity_fp.py

```diff
@@ -84,85 +94,85 @@ def test_add_liquidity_with_long_immediately(self):
         market_deltas, wallet_deltas = self.hyperdrive.add_liquidity(self.bob.wallet, self.contribution)
 
         # Ensure that the contribution was transferred to Hyperdrive.
-        self.assertEqual(market_deltas.d_base_asset, -wallet_deltas.balance.amount)
+        self.assertEqual(float(market_deltas.d_base_asset), float(-wallet_deltas.balance.amount))
 
         # Ensure that the new LP receives the same amount of LP shares as the initializer.
-        self.assertAlmostEqual(market_deltas.d_lp_total_supply, lp_supply_before, 6)
-        self.assertEqual(self.hyperdrive.market_state.lp_total_supply, lp_supply_before * 2)
+        self.assertAlmostEqual(float(market_deltas.d_lp_total_supply), float(lp_supply_before), places=6)
```

Comment:
you mean using `int` & `delta` instead of `float` and `places`? I guess they are interchangeable. `FixedPoint(48)` and `FixedPoint(49)` have delta of 1, and places would be 18. If it is larger, e.g. `FixedPoint(948)` and `FixedPoint(949)`, it also has delta of 1 and places of 18. Right? I don't care either way. I think `AlmostEqual` is using `round` under the hood so it's losing any float noise anyways? I think that's what you're saying when you say they get converted to the same float.